libs/designer/src/lib/core/index.ts

```diff
@@ -9,6 +9,7 @@ export type { NodeInputs } from './state/operation/operationMetadataSlice';
 export { useOperationsInputParameters } from './state/operation/operationSelector';
 export { discardAllChanges, setFocusNode, setIsGraphLocked, setIsWorkflowDirty } from './state/workflow/workflowSlice';
 export { useIsWorkflowDirty, useNodeDisplayName } from './state/workflow/workflowSelectors';
+export { useIsDesignerDirty, resetDesignerDirtyState } from './state/global';
```

Comment:
Could you also export `useIsWorkflowParametersDirty`? I have a feeling we may want to consume that.

Reply:
Yep I can do that 👍